### PR TITLE
fix glm.

### PR DESCRIPTION
**Pull request by hkim89**
Susie-glm is fixed now.  Will work on the null model and prepare the tutorial.

### CODE REVIEW

**Comment by gaow** [comment]
@hkim89 sounds great! Could you summarize at the high level what was wrong, and also point to the relevant lines of code in this patch for what was fixed to correct it?

**Comment by hkim89** [comment]
The initialization for \alpha  & b_l's for update in the em algorithm needs to use 'copy()' rather than 'zeros()' in order for the current updates not to be affected by the previous ones acting as pointers  (A1,B1) on line 437 in VEM.jl.  I put them yesterday and check today with all parameter updates (\xi, \beta, etc. ) using 'copy()' on line 470.   Also checked with julia simulation code on the bottom in scratch.jl transferred from R version by Andrew and then made consistent result.

**Comment by gaow** [comment]
I see the problem now. My complaint is that I don't see `copy()` as an elegant approach ... there has to be more elegant ways without having to copy the data around?
